src/pages/EnvironmentsPage/environmentsPermissions.ts

```diff
@@ -8,19 +8,19 @@ import { getTerraUser } from 'src/libs/state';
 
 const makePermissionsProvider = (userEmailGetter: () => string): LeoResourcePermissionsProvider => {
   return <LeoResourcePermissionsProvider>{
-    canDeleteDisk: (disk: PersistentDisk) => {
+    hasDeletePermission: (disk: PersistentDisk) => {
       const currentUserEmail = userEmailGetter();
       return disk.auditInfo.creator === currentUserEmail;
     },
-    canPauseResource: (resource: App | ListRuntimeItem) => {
+    hasPausePermission: (resource: App | ListRuntimeItem) => {
       const currentUserEmail = userEmailGetter();
       const creator = getCreatorForCompute(resource);
       return currentUserEmail === creator;
     },
-    canDeleteApp: (resource: App) => {
+    isAppInDeletableState: (resource: App) => {
```

Comment:
do we never check if a user has permission to delete as with disks? `isResourceDeletable` only checks if that status is deletable AFAICT

Reply:
No... the delete flow is a little wonky for sure in terms of permission/checking if resource is deletable. This isn't our business logic though, this was added by other teams. I'd rather campsite rule this in terms of making the interface more clear, and defer on updating any logic since this is fixing a logic bug to begin with. 

Honestly, the `environmentPermissions` component is already overloaded and should not be in the business of resource state checking beyond permissions IMO. Notice how in the `PauseButton` the button itself has the logic about whether the resource is pausable, as well as the associated utility method.

Reply:
yeahhh 😬  its probably out of the scope of this fix, just something I noticed